samcli/commands/_utils/options.py

```diff
@@ -594,7 +594,13 @@ def remote_invoke_parameter_click_option():
         type=RemoteInvokeBotoApiParameterType(),
         callback=remote_invoke_boto_parameter_callback,
         required=False,
-        help="Additional parameters for the boto API call.\n" "Lambda APIs: invoke and invoke_with_response_stream",
+        help="Additional parameters that can be passed to the boto API call to invoke the resource.\n"
```

Comment:
the customer should not know about the implementation details such as `boto`. we should remove that.